tools/packaging/packaging.strings.psd1

```diff
@@ -48,6 +48,12 @@ case "$1" in
         remove-shell "{0}"
         ;;
 esac
+'@
+
+    MacOSAfterInstallScript = @'
+#!/bin/bash
```

Comment:
We should do something more like the redhat script so that we don't create duplicates.